apps/docs/src/app/platform/component-docs/platform-button/platform-button-docs.component.html

```diff
@@ -0,0 +1,106 @@
+<fd-docs-section-title [id]="'Button-options'" [componentName]="'button'"> Button Options </fd-docs-section-title>
+<description>
+    There are three emphasis styles used to indicate the importance of the button on the page. Do not provide a value
+    for the default option. Use <code class="code-snippet">[options]="'emphasized'"</code> or
+    <code class="code-snippet">[options]="'light'"</code> to set the option.
+</description>
+<component-example [name]="'ex0'">
+    <fdp-button-options-example></fdp-button-options-example>
+</component-example>
+<code-example [exampleFiles]="buttonHtmlOptions"></code-example>
+<separator></separator>
+
+<fd-docs-section-title [id]="'Button-types'" [componentName]="'button'"> Button Types </fd-docs-section-title>
+<description>
+    The default button type is "action", no modifier is required. Use
+    <code class="code-snippet">[type]="'standard'", [type]="'positive'", [type]="'medium'"</code>, or
+    <code class="code-snippet">[type]="'negative'"</code> to modify the type of the button.
+</description>
+<component-example [name]="'ex1'">
+    <fdp-button-types-example></fdp-button-types-example>
+</component-example>
+<code-example [exampleFiles]="buttonHtmlType"></code-example>
+<separator></separator>
+
+<fd-docs-section-title [id]="'Button-sizes'" [componentName]="'button'"> Button Sizes </fd-docs-section-title>
+<description> Use <code class="code-snippet">[compact]="true"</code> for the mobile button size. </description>
+<component-example [name]="'ex2'">
+    <fdp-button-sizes-example></fdp-button-sizes-example>
+</component-example>
+<code-example [exampleFiles]="buttonHtmlSize"></code-example>
+<separator></separator>
+
+<fd-docs-section-title [id]="'Button-icons'" [componentName]="'button'">
+    Button with Icons
+</fd-docs-section-title>
+<description>
+    Use
+    <code class="code-snippet">[glyph]="'{{ '{' }}icon-name}'"</code> to add an icon to the button.
+</description>
+<component-example [name]="'ex3'">
+    <fdp-button-icons-example></fdp-button-icons-example>
+</component-example>
+<code-example [exampleFiles]="buttonHtmlIcon"></code-example>
+<separator></separator>
+
+<fd-docs-section-title [id]="'Button-state'" [componentName]="'button'">
+    Button State
+</fd-docs-section-title>
+<description>
+    'disabled' true makes the button disabled when we make it false it won't be disabled any more, by default disabled
+    is false
```

Comment:
Can you change this sentence to:
<code>[disabled]="true"</code> makes the button disabled, when we make it false, it won't be disabled any more, by default disabled is set to false.